File: 125-valid-palindrome/valid-palindrome.java
class Solution {
    public boolean isPalindrome(String s) {
        s=s.toLowerCase();
        s=s.replaceAll("[^a-zA-Z0-9]","");
        
        int j=0;
        int h=s.length()-1;
        while(j<h){
            if(s.charAt(j)!=s.charAt(h)){
                return false;
            }
            j++;
            h--;
        }
        return true;

    }
}

File: 125-valid-palindrome/valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        s=s.lower()
        rev=""
        for i in s:
            if 'a'<=i<='z' or '0'<=i<='9':
                rev+=i
        if rev==rev[::-1]:
            return True
        else:
            return False
        